x/txreward/client/cli/tx.go

```diff
@@ -0,0 +1,53 @@
+package cli
+
+import (
+	"github.com/commercionetwork/commercionetwork/x/txreward/internal/types"
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/client/context"
+	"github.com/cosmos/cosmos-sdk/codec"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/x/auth"
+	"github.com/cosmos/cosmos-sdk/x/auth/client/utils"
+	"github.com/spf13/cobra"
+)
+
+func GetTxCmd(cdc *codec.Codec) *cobra.Command {
+	txCmd := &cobra.Command{
+		Use:                        types.ModuleName,
+		Short:                      "CommercioTXREWARDS transactions subcommands",
```

Comment:
I think there's a typo

```suggestion
		Short:                      "CommercioTBR transactions subcommands",
```